core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
```

Comment:
The difference between `PRIVMSG` and `NOTICE` is whether bots should reply to the message.
Should reactions be as `PRIVMSG` or `NOTICE`?

I prefer @SaberUK 's new `METAMSG` verb and not to change syntax of widely used `PRIVMSG`/`NOTICE`